src/game/editor/editor.h

```diff
@@ -147,6 +147,13 @@ class CLayer
 	virtual void ModifyEnvelopeIndex(INDEX_MODIFY_FUNC pfnFunc) {}
 	virtual void ModifySoundIndex(INDEX_MODIFY_FUNC pfnFunc) {}
 
+	virtual CLayer *Duplicate() { return new CLayer(*this); }
+	void CopyTo(CLayer *pLayer) {
+		mem_copy(pLayer->m_aName, m_aName, 12);
```

Comment:
```suggestion
		mem_copy(pLayer->m_aName, m_aName, sizeof(pLayer->m_aName));
```

I first thought it was a string, but it's actually an int array.

Reply:
Yeah I just changed that in my last commit